rootfs/rootfs/etc/rc.d/prltoolsd

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+/bin/sh /usr/local/etc/init.d/prltoolsd start
```

Comment:
I'm definitely down with that if it works in Linux, but otherwise I'll just
trust Sven's testing. :smile: :wink: